doc/releases/release-notes-3.4.rst

```diff
@@ -379,6 +379,8 @@ Libraries / Subsystems
 * File systems
 
   * Added :kconfig:option:`CONFIG_FS_FATFS_REENTRANT` to enable the FAT FS reentrant option.
+  * With LittleFS as backend, :c:func:`fs_mount` return code was changed to ``EFAULT`` when
+    called with ``FS_MOUNT_FLAG_NO_FORMAT`` and the FS is damaged.
```

Comment:
"and the FS is damaged." change this to something line "while designated LIttleFS area could not be mounted because it has not yet been mounted or required reformatting" ( avoid damaged).